examples/problem_decomposition/mifno.ipynb

```diff
@@ -299,35 +314,36 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/home/alex/VirtEnvs/qsdk/lib/python3.10/site-packages/pyscf/dft/libxc.py:772: UserWarning: Since PySCF-2.3, B3LYP (and B3P86) are changed to the VWN-RPA variant, the same to the B3LYP functional in Gaussian and ORCA (issue 1480). To restore the VWN5 definition, you can put the setting \"B3LYP_WITH_VWN5 = True\" in pyscf_conf.py\n",
+      "/home/alex/.pyenv/versions/qsdk/lib/python3.10/site-packages/pyscf/dft/libxc.py:772: UserWarning: Since PySCF-2.3, B3LYP (and B3P86) are changed to the VWN-RPA variant, the same to the B3LYP functional in Gaussian and ORCA (issue 1480). To restore the VWN5 definition, you can put the setting \"B3LYP_WITH_VWN5 = True\" in pyscf_conf.py\n",
```

Comment:
let's remove that. If you run the cell a second time right away, the warning should not appear again

Reply:
I did not know that, it worked thanks.